CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.13.85](https://github.com/brave/brave-browser/releases/tag/v1.13.85)
+
+ - Implemented cross-platform promotion modal. ([#10571](https://github.com/brave/brave-browser/issues/10571))
+ - Implemented "Refresh Status" button on rewards panel to check for publisher status. ([#10005](https://github.com/brave/brave-browser/issues/10005))
+ - Implemented version 2 of onboarding. ([#9555](https://github.com/brave/brave-browser/issues/9555))
```

Comment:
Probably can word this a bit better. User has no idea what "version 2" means. Maybe something along the lines of the following:

* `Implemented new version of onboarding`